### DISCUSSION TITLE
I2C Sensor

### DISCUSSION THREAD

**Discussion by Ninja1283** [arendst/Tasmota#11703] [Q&A] [answered]
I am trying to use a few Adafruit SHT40 temp/humidity sensors with Tasmota. I have the GPIO pins configured for clock and data, and am using 9.3.1(sensors) firmware. i2cscan shows the sensor @ 0x44, but will not display any info.

I didn't see the SHT40 sensor on the Supported I2C device list, just the older 1x and 3x devices. Do I need to recompile to use this sensor?
General Info [here](https://learn.adafruit.com/adafruit-sht40-temperature-humidity-sensor). Libraries [here](https://adafruit.github.io/Adafruit_SHT4X/html/index.html).

**Comment by sfromis** [comment, marked as answer]
As SHT40 is a new sensor type, it looks like no one yet has come forward to create or amend a driver for it. Commonly, sensor support is added by someone with the interest doing a driver and submitting a PR.

**Comment by Jason2866** [comment]
Only the exact type of the listed sensor is supported. Every sensor needs a specific driver.
